nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core-api/src/main/java/org/apache/nifi/controller/AbstractConfiguredComponent.java

```diff
@@ -57,6 +57,7 @@ public AbstractConfiguredComponent(final ConfigurableComponent component, final
         this.component = component;
         this.validationContextFactory = validationContextFactory;
         this.serviceProvider = serviceProvider;
+        this.name = new AtomicReference<>(component.getClass().getSimpleName());
```

Comment:
That's kind of how it was and yes, this is me trying to "extract an answer by force" ;). I don't see a reason why it would have to be initialized to null while defaulting to class name in sub-class, but I also do see a "safety" point in your comment. I am good either way. Do you think we need @markap14  or anyone else to chip in with opinion?